From: Ben Hutchings <ben@decadent.org.uk>
Date: Tue, 16 Sep 2014 02:51:20 +0100
Subject: SCSI: Avoid ABI change in 3.2.62

Commit b14bf2d0c035 ('usb-storage/SCSI: Add broken_fua blacklist flag')
adds a new 1-bit field to struct scsi_device.  This is fine because it
had 29 bits of padding to spare, but we need to hide the new field
from genksyms.

--- a/include/scsi/scsi_device.h
+++ b/include/scsi/scsi_device.h
@@ -151,7 +151,10 @@ struct scsi_device {
 	unsigned no_read_disc_info:1;	/* Avoid READ_DISC_INFO cmds */
 	unsigned no_read_capacity_16:1; /* Avoid READ_CAPACITY_16 cmds */
 	unsigned is_visible:1;	/* is the device visible in sysfs */
+#ifndef __GENKSYMS__
 	unsigned broken_fua:1;		/* Don't set FUA bit */
+	/* 28 bits of padding */
+#endif
 
 	DECLARE_BITMAP(supported_events, SDEV_EVT_MAXBITS); /* supported events */
 	struct list_head event_list;	/* asserted events */
